### PR TITLE
chore: update flake-parts to wrap std

### PR DESCRIPTION
**Pull request by bernokl**
# Context

_Reason for the change? If an issue exists, reference it here using a [keyword](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)_

# Proposed Solution

# Important Changes Introduced

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {
   inputs = {
     nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
+		flake-parts.url = "github:hercules-ci/flake-parts";
```
Comment (michalrus) [inline flake.nix:4]: Missing `treefmt`, these are actual tab (`\t`) characters =)

**Review by michalrus — COMMENTED** [review]
One nitpick, otherwise LGTM

**Comment by michalrus** [comment]
Also there’s a [failing action](https://github.com/input-output-hk/cardano-js-sdk/actions/runs/8082785641) `workflow_dispatch:`’ed by @gytis-ivaskevicius 

The [same action run automatically](https://github.com/input-output-hk/cardano-js-sdk/actions/runs/8081778114) by @bernokl worked

But it’s most probably unrelated to these changes, I’ll fix that in a separate PR :memo:

**Review by gytis-ivaskevicius — APPROVED** [review]

**Review by michalrus — APPROVED** [review]
lgtm

**Pull request merged**